src/optimagic/optimizers/iminuit_migrad.py

```diff
@@ -33,19 +32,30 @@
 @dataclass(frozen=True)
 class IminuitMigrad(Algorithm):
     stopping_maxfun: int = STOPPING_MAXFUN
-    stopping_maxiter: int = STOPPING_MAXITER
-    errordef: Optional[float] = None
 
     def _solve_internal_problem(
         self, problem: InternalOptimizationProblem, params: NDArray[np.float64]
     ) -> InternalOptimizeResult:
         def wrapped_objective(x: NDArray[np.float64]) -> float:
             return float(problem.fun(x))
 
-        m = Minuit(
-            wrapped_objective,
-            params,
-        )
+        wrapped_gradient = None
+        jac_func = None
+        if problem.jac is not None:
```

Comment:
You don't need this check. `problem.jac` is never None. If the user did not pass a derivative, it is a numerical one.